Documentation/contributing/development/dev_setup.rst

```diff
@@ -57,33 +57,39 @@ Version Requirements
++ `ginkgo <https://github.com/onsi/ginkgo>`__                       | >= 1.4.0 and < 2.0.0         | ``go install github.com/onsi/ginkgo/ginkgo@v1.16.5``            |
++-------------------------------------------------------------------+------------------------------+-----------------------------------------------------------------+
++ `golangci-lint <https://github.com/golangci/golangci-lint>`_      | >= v1.27                     | N/A (OS-specific)                                               |
++-------------------------------------------------------------------+------------------------------+-----------------------------------------------------------------+
++ `cfssl <https://github.com/cloudflare/cfssl>`_                    | >= v1.6.0                    | ``go install github.com/cloudflare/cfssl/cmd/cfssl@latest``     |
++-------------------------------------------------------------------+------------------------------+-----------------------------------------------------------------+
++ `cfssljson <https://github.com/cloudflare/cfssl>`_                | >= v1.6.0                    | ``go install github.com/cloudflare/cfssl/cmd/cfssljson@latest`` |
++-------------------------------------------------------------------+------------------------------+-----------------------------------------------------------------+
++ `Docker <https://docs.docker.com/engine/installation/>`_          | OS-Dependent                 | N/A (OS-specific)                                               |
++-------------------------------------------------------------------+------------------------------+-----------------------------------------------------------------+
++ `Docker-Compose <https://docs.docker.com/compose/install/>`_      | OS-Dependent                 | N/A (OS-specific)                                               |
++-------------------------------------------------------------------+------------------------------+-----------------------------------------------------------------+
++ python3-pip                                                       | latest                       | N/A (OS-specific)                                               |
++-------------------------------------------------------------------+------------------------------+-----------------------------------------------------------------+
++ `helm <https://helm.sh/docs/intro/install/>`_                     | >= v3.6.0                    | N/A (OS-specific)                                               |
++-------------------------------------------------------------------+------------------------------+-----------------------------------------------------------------+
++ `kind <https://kind.sigs.k8s.io/docs/user/quick-start/>`_         | >= v0.7.0                    | ``go install sigs.k8s.io/kind@v0.19.0``                         |
```

Comment:
because of this blocker issue, https://github.com/kubernetes-sigs/kind/issues/3277, we would not want to recommend latest version but `v0.19.0`.

Reply:
Can you add an extra underscore to the end of the link here? The netifly preview is failing to build because there are now two external links that both use the name `kind`:

* netifly build: https://app.netlify.com/sites/docs-cilium-io/deploys/6512163e3b428b0009ea37ab

I had to do some Googling to figure this out, so for reference here are some helpful resources I found on this:

* Explanation: https://github.com/sphinx-doc/sphinx/issues/3921
* StackOverflow reference on `_` vs `__`: https://stackoverflow.com/questions/74252545/differences-between-anonymous-and-non-anonymous-hyperlinks-in-sphinx

```suggestion
+ `kind <https://kind.sigs.k8s.io/docs/user/quick-start/>`__         | >= v0.7.0                    | ``go install sigs.k8s.io/kind@v0.19.0``                         |
```

Reply:
I see thanks for sharing the details.

```bash
4:23:33 PM: /opt/build/repo/Documentation/contributing/development/dev_setup.rst:11: WARNING: Duplicate explicit target name: "kind".
```

https://github.com/sphinx-doc/sphinx/issues/3921#issuecomment-315581557 addresses the issue.

addressed by https://github.com/cilium/cilium/pull/28269/commits/8de709367fb5369b173e0504112126265f332f91